test/Btn.spec.js

```diff
@@ -0,0 +1,57 @@
+import { Btn } from '../src/app/components/Btn';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+
+describe('Function that creates and renders button', () => {
```

Comment:
Brawo za testing-library! Bo już miałęm się przyczepiać do testów :) 

Wystarczy chyba Button :) W opisie testach skupiajmy się na zachowaniu, czyli niezbyt obchodzi nas, że jest to funkcja czy klasa. Ważne, że to przycisk, który robi coś tam co później opisujemy.